package-system/AWSNativeSDK/build_AWSNativeSDK_linux.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+echo "Install required library: zlib1g-dev"
+sudo apt install -y zlib1g-dev || (echo "Install required library: zlib1g-dev failed" ; exit 1)
+
+echo "Install required library: libssl-dev"
+sudo apt install -y libssl-dev || (echo "Install required library: libssl-dev failed" ; exit 1)
+
+echo "Install required library: libcurl4-openssl-dev"
+sudo apt install -y libcurl4-openssl-dev || (echo "Install required library: libcurl4-openssl-dev failed" ; exit 1)
```

Comment:
I dont think we should be sudo-ing in any of these scripts - we intend this to run automatically and unattended at some point, so we can't require admin

a better way (on ubuntu) is to just check if its installed instead, which does NOT require sudo
```
if dpkg-query -W -f'${Status}' "zlib1g-dev" 2>/dev/null | grep -q "ok installed"; then
 echo Required package zlib1g-dev is not installed.
 exit 1
fi;
```

But I think its probably even better not to check and allow the build script to fail.
Maybe print at the top "you need zlib1g-dev, libssl-dev, and libcurl4-openssl-dev in order to compile this"

This is so that if you're not on ubuntu (like you're on redhat) you still have a chance to satisfy its requirements and compile it anyway

Reply:
sounds good, will do